<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Szyfrowanie 2DLMCA</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushLoadButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Załaduj obraz</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushDecryptButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Deszyfruj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushEncryptButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Szyfrowanie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parametr &amp;quot;k&amp;quot; odpowiada za liczbę utworzonych szyfrogramów konfiguracyjnych.&lt;/p&gt;&lt;p&gt;k = 0 odpowiada oryginalnemu obrazowi.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parametr &amp;quot;k&amp;quot; odpowiada za liczbę utworzonych szyfrogramów konfiguracyjnych.&lt;/p&gt;&lt;p&gt;k = 0 odpowiada oryginalnemu obrazowi.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>parametr k:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="originalImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ustaw jak ma być traktowany obraz - jako obraz kolorowy, szary lub binarny.&lt;/p&gt;&lt;p&gt;Pamiętaj również zaznaczyć tę opcję przy wykonywaniu procesu deszyfracji !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Ustaw rodzaj obrazu:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="areaImageType">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioImageColor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>kolorowy</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioImageGrey">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>szary</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioImageBinary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>binarny</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parametr &amp;quot;m&amp;quot; odpowiada ilości wykonanych dodatkowych iteracji.&lt;/p&gt;&lt;p&gt;Pamiętaj, że musi być:&lt;span style=&quot; font-style:italic;&quot;&gt; m &amp;gt;= k&lt;/span&gt; !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parametr &amp;quot;m&amp;quot; odpowiada ilości wykonanych dodatkowych iteracji.&lt;/p&gt;&lt;p&gt;Pamiętaj, że musi być:&lt;span style=&quot; font-style:italic;&quot;&gt; m &amp;gt;= k&lt;/span&gt; !&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>parametr m:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxM">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Parametr &amp;quot;n&amp;quot; odpowiada za liczbę utworzonych szyfrogramów do rozdzielenia.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>parametr n:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxN">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>547</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushLoadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>51</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushEncryptButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>encryption()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadFile()</slot>
  <slot>encryption()</slot>
 </slots>
</ui>
